pkg/cmd/admin/groups/sync/cli/sync.go

```diff
@@ -103,6 +103,14 @@ type SyncOptions struct {
 	Out io.Writer
 }
 
+// CreateErrorHandler creates an error handler for the LDAP sync job
+func (o *SyncOptions) CreateErrorHandler() syncerror.Handler {
+	return syncerror.NewCompoundHandler(
```

Comment:
NewMemberLookupOutOfBoundsHandler doesn't need to take a boolean... just don't include it if `o.Config.TolerateOutOfScopeMembers` is false